tl/appendices/3-0-migration-guide.rst

```diff
@@ -1,57 +1,60 @@
-- CakePHP 3.x requires the mbstring extension.
-- CakePHP 3.x requires the intl extension.
+- Ang CakePHP 3.x  ay sumusuporta sa PHP Bersyon 5.4.16 at pataas.
+- Ang CakePHP 3.x ay nangangailangan ng mbstring na ekstensiyon.
+- Ang CakePHP 3.x ay nangangailangan ng intl na ekstensiyon.
 
-.. warning::
+.. babala::
 
-    CakePHP 3.0 will not work if you do not meet the above requirements.
+	Ang CakePHP 3.0 ay hindi gagana kung hindi mo nakamit ang mga kinakailangan sa itaas.
 
-Upgrade Tool
-============
+Upgrade na Kasangkapan
+======================
 
-While this document covers all the breaking changes and improvements made in
-CakePHP 3.0, we've also created a console application to help you complete some
-of the time consuming mechanical changes. You can `get the upgrade tool from
+Habang ang dokumentong ito ay tumatalakay sa lahat ng nakakasirang mga pagbabago at
+mga pagpapabuti na ginawa sa CakePHP 3.0, kami ay nakalikha rin ng isang console na
+aplikasyon upang tumulong sa iyo na kumpletuhin ang ilang nakakaubos ng oras na mekanikal
+na mga pagbabago. Maaari kang `kumuha sa pang-upgrade na kasangkapan mula sa 
 github <https://github.com/cakephp/upgrade>`_.
 
-Application Directory Layout
-============================
+Layout ng Aplikasyon na Direktoryo
+==================================
 
-The application directory layout has changed and now follows
-`PSR-4 <http://www.php-fig.org/psr/psr-4/>`_. You should use the
-`app skeleton <https://github.com/cakephp/app>`_ project as a reference point
-when updating your application.
+Ang layout ng aplikasyon na direktoryo ay nabago at ngayon ay sumusunod sa
+`PSR-4 <http://www.php-fig.org/psr/psr-4/>`_. Dapat mong gamitin ang 
+`app skeleton <https://github.com/cakephp/app>`_ na proyekto bilang isang 
+punto ng reperensya kapag nag-a-update ng iyong aplikasyon.
 
-CakePHP should be installed with Composer
-=========================================
+Ang CakePHP ay dapat naka-install gamit ang Composer
+====================================================
 
-Since CakePHP can no longer be installed via PEAR, or in a shared
-directory, those options are no longer supported. Instead you should use
-`Composer <http://getcomposer.org>`_ to install CakePHP into your application.
+Dahil ang CakePHP ay hindi na nai-install gamit ang PEAR, o sa isang nakabahaging
+direktoryo, ang mga opsyon na iyon ay hindi na suportado. Sa halip dapat mong 
+gamitin ang `Composer <http://getcomposer.org>`_ upang i-install ang CakePHP sa
+iyong aplikasyon.
 
-Namespaces
-==========
+Mga Namespace
+=============
 
-All of CakePHP's core classes are now namespaced and follow PSR-4 autoloading
-specifications. For example **src/Cache/Cache.php** is namespaced as
-``Cake\Cache\Cache``.  Global constants and helper methods like :php:meth:`__()`
-and :php:meth:`debug()` are not namespaced for convenience sake.
+Lahat ng core na mga class ng CakePHP ay naka-namespace na ngayon ay sumusunod sa
```

Comment:
na ngayon at sumusunod

Reply:
Much better if we have something like this:
CakePHP ay kasalukuyang naka-namespace at sumusunod sa

Reply:
That sounded much better. Thank you for pointing that out.